drivers/SmartThings/zwave-switch/fingerprints.yml

```diff
@@ -923,13 +923,6 @@ zwaveGeneric:
     deviceLabel: Dimmer Switch
     genericType: 0x11 # GENERIC_TYPE_SWITCH_MULTILEVEL
     deviceProfileName: switch-level
-  - id: "GenericDimmerSwitch/3"
```

Comment:
Why was this generic fingerprint removed?

Reply:
that was not intentional!